tiingo/README.md

```diff
@@ -0,0 +1,29 @@
+# Chainlink External Adapter for Tiingo
+
+## Configuration
+
+This adapter supports the following environment variables:
+
+- `API_KEY`: Your Tiingo API key
+
+## Input Params
+
+- `ticker`: The stock ticker to query
```

Comment:
Since this is a required param could we mark it as such?

Trying out a new README template that might be a little more clear:
https://github.com/smartcontractkit/external-adapters-js/blob/develop/example/README.md